src/install/python/python-frameworks/python-agent-scgi.mdx

```diff
@@ -0,0 +1,19 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using SCGI
+---
+
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command.
```

Comment:
```suggestion
You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command. There are no specific steps you need to take to integrate with SCGI so skip straight to the server specific instructions.
```